### PR TITLE
Create graphql node for creating new organization

### PR DESCRIPTION
**Pull request by sudan45**
Addresses 
 - #1444 

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] n+1 queries
- [x] flake8 issues
- [x] `print`
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [x] tests
- [ ] permission checks (tests here too)
- [ ] translations

### CODE REVIEW

File: apps/organization/mutation.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+class OrganizationCreate(GrapheneMutation):
+    class Arguments:
+        data = OrganizationInputType(required=True)
+    model = Organization
+    result = graphene.Field(OrganizationType)
+    serializer_class = OrganizationGqSerializer
+
+    @classmethod
+    def check_permissions(cls, info, **kwargs):
+        return True
```
Comment (susilnem) [inline apps/organization/mutation.py]: Can we verify this permission? @sudan45

File: apps/organization/serializers.py [hunk 2]
```diff
@@ -69,3 +69,10 @@ class Meta:
         model = Organization
         fields = ('key', 'title', 'long_name',
                   'short_name', 'logo', 'organization_type', 'merged_as')
+
+
+class OrganizationGqSerializer(UserResourceSerializer):
+    class Meta:
+        model = Organization
+        # fields = ('url','organization_type','title','short_name')
```
Comment (susilnem) [inline apps/organization/serializers.py]: Remove the comment line.

File: apps/organization/mutation.py (hunk 1 above)
Comment (thenav56) [reply apps/organization/mutation.py]: In deep, all users have ability to create Organizations if needed.
Each organization have `verified` field, which here is set to False, From admin panel, maintainers can verify/merge organizations as required.

**Review by thenav56 — APPROVED** [review]

File: apps/organization/tests/test_mutations.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        super().setUp()
+        self.member_user = UserFactory.create()
+
+    def test_create_organization(self):
+        def _query_check(minput, **kwargs):
+            return self.query_check(
+                self.CREATE_ORGANIZATION,
+                minput=minput,
+                **kwargs
+            )
```
Comment (thenav56) [inline apps/organization/tests/test_mutations.py]: ?

**Review by thenav56 — CHANGES_REQUESTED** [review]

File: apps/organization/tests/test_mutations.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+    def setUp(self):
+        super().setUp()
+        self.member_user = UserFactory.create()
+
+    def test_create_organization(self):
+        def _query_check(minput, **kwargs):
+            return self.query_check(
+                self.CREATE_ORGANIZATION,
+                minput=minput,
```
Comment (k9845) [inline apps/organization/tests/test_mutations.py:35]: where is `minput ` data?

**Review by susilnem — COMMENTED** [review]
LGTM!

File: apps/organization/tests/test_mutations.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+        # without login
+        _query_check(minput, assert_for_error=True)
+
+        # with login
+
+        self.force_login(user)
+
+        content = _query_check(minput)
+        self.assertEqual(content['data']['organizationCreate']['errors'], None)
+        self.assertEqual(content['data']['organizationCreate']['result']['title'], 'Test Organization')
```
Comment (thenav56) [inline apps/organization/tests/test_mutations.py:47]: Let's also check the is_verified field. Which should be false.

**Review by thenav56 — CHANGES_REQUESTED** [review]
LGTM

**Review by thenav56 — APPROVED** [review]

**Pull request merged**